design/quarterly-release.md

```diff
@@ -0,0 +1,49 @@
+# Change Contour releases to quarterly
+
+Status: Accepted
+
+## Abstract
+Move Contour and Contour Operator to a longer release cadence to cushion for release planning, align Contour releases with Contour Operator releases, and align with downstream VMware TKG releases, and improve feature planning. 
+
+Please see [#3634](https://github.com/projectcontour/contour/issues/3634) for more discussion.
+
+## Current issues with monthly release
+* Very tough for users to keep up with Contour monthly releases. It’s rare for production deployments to always chase the latest version and more likely to upgrade every 6 months or longer. As part of these changes, we will be investigating how possible enabling jump upgrades is for Contour.
+* Release engineering is taxing and we have to spend some time preparing for this every month. The tradeoff here is that we will need to do more patch releases, and that quarterly releases will take more effort to put together than monthly ones, simply because they'll be bigger.
+* Since we release the Contour Operator 1:1 with Contour, the Operator release adds to the complexity of release engineering.  An issue with a Contour release will thus affect Contour Operator pushing it out and in turn eat into the next release cycle
```

Comment:
I still think it's worth a discussion to move the operator logic into a new command in Contour. That would centralize everything and remove the confusing around releasing both in step.

Reply:
+1 on this for me

Reply:
Agreed, but I think we'll document this as things stand today, and revise if and when we change where the operator is stored.